.changeset/sweet-moose-wave.md

```diff
@@ -0,0 +1,6 @@
+---
+"@finos/legend-extension-external-store-service": patch
+"@finos/legend-manual-tests": patch
+---
+
```

Comment:
Can you please add a new changeset for `@finos/legend-manual-tests` without any description. We tend to only add description if it changes the test functionally, not when we add new test case

Reply:
done